### DISCUSSION TITLE
Rick Roll?

### DISCUSSION THREAD

**Discussion by jbsiddall** [tailwindlabs/tailwindcss#13679] [Help]
![image](https://github.com/tailwindlabs/tailwindcss/assets/1850562/7ced9088-13d5-4dd6-a57f-9fa24e526ce8)

Umm am i missing the joke or is tailwind website been hacked with rich roll?
https://tailwindcss.com/docs/aspect-ratio

**Comment by wongjn** [comment]
I think that page has always been like that. It's showcasing the `aspect-video` class name. So this class name is being demonstrated with a video. Perhaps the video was chosen for some comedic value.

**Comment by jbsiddall** [comment]
Thanks @wongjn! In that case i'll go appreciate the rick roll and its delightful aspect ratio
